kmath-commons/src/main/kotlin/space/kscience/kmath/commons/linear/CMMatrix.kt

```diff
@@ -101,8 +101,8 @@ public object CMMatrixContext : MatrixContext<Double, CMMatrix> {
     public override operator fun Matrix<Double>.minus(b: Matrix<Double>): CMMatrix =
         CMMatrix(toCM().origin.subtract(b.toCM().origin))
 
-    public override fun multiply(a: Matrix<Double>, k: Number): CMMatrix =
-        CMMatrix(a.toCM().origin.scalarMultiply(k.toDouble()))
+//    public override fun multiply(a: Matrix<Double>, k: Number): CMMatrix =
```

Comment:
Why does not MatrixContext implement ScaleOperations?

Reply:
It is a good question, but Matrix does not know about numbers, it knows only about T.

Reply:
I'll add it to numeric matrices though.